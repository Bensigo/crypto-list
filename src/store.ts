import { createStore, applyMiddleware } from "redux";
import { composeWithDevTools } from "remote-redux-devtools";
import createSagaMiddleware from "redux-saga";

import reducer from "./reducers";
import rootSaga from "./sagas";

const sagaMiddleware = createSagaMiddleware();

const middlewareWithDevTool =  composeWithDevTools(
    applyMiddleware(sagaMiddleware)
);
const store = createStore(reducer, middlewareWithDevTool);

sagaMiddleware.run(rootSaga);
export default store;