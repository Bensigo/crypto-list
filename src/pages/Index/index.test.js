import React from "react"
import renderer from "react-test-renderer"
import {Provider} from "react-redux"
import store from "../../store"
import List from "./index.tsx"


describe("List", () => {

    it("render component", () => {
       const component = renderer.create(
           <Provider store={store}>
               <List></List>
           </Provider>
       );
       const tree = component.toJSON();
       expect(tree).toMatchSnapshot()
    })
})