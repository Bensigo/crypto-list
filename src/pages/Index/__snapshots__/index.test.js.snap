// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List render component 1`] = `
<div
  className="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
  style={
    Object {
      "height": "70vh",
    }
  }
>
  <p
    className="MuiTypography-root MuiTypography-body1"
  />
</div>
`;
